README.md

```diff
@@ -236,7 +210,7 @@ Please ensure that your tests still run before project submission, of course. :)
 
 ### Optional: Wave 5
 
-Change the `Adagrams` object into a class named `Adagrams`!
+Encapsulate the functions in a class named `Adagrams`!
```

Comment:
Do we still want to even mention this? Because the functions aren't in anything, the way they would get called will not be "just compatible" with the existing tests. So if the tests are still supposed to pass, they would need to make changes to how the tests get the functions. And on submission, the tests are likely to fail (if we follow the usual track of replacing the submission's tests).

If we'd like students to think about an OOP approach, maybe ask them to work in a separate branch? But even still they'd have to do research about how to change the exporting. We don't actually talk about exporting and anything module-related until React, do we?

Reply:
Ok. This should be fine. If they want to work on the optional stuff, they might need to ask for additional clarification, but that's up to them.